department-of-veterans-affairs/va.gov-cms
# VACMS-3332: Add logic to toggle standalone qa entity view display.

## Description
See #3332 

## Testing done
Visual / behat

## Screenshots
**Enabled:**
![image](https://user-images.githubusercontent.com/2404547/98703355-6809f580-2349-11eb-86f5-bab0844a074e.png)

**Disabled:**
![image](https://user-images.githubusercontent.com/2404547/98703368-6c361300-2349-11eb-89ad-ea2176f8d629.png)

## QA steps
- As an administrator, go to `node/8241/edit` and toggle off standalone display, and save the node.
- Visually verify that latest revision display looks like the disabled screenshot.
- Go back to edit page, toggle on standalone display, and save the node.
- Visually verify that the latest revision display looks like the enabled screenshot.
